src/templates/partials/caddy.hbs

```diff
@@ -1,43 +1,31 @@
 # {{output.header}}
 # {{{output.link}}}
-{{#unless (includes "old" form.config)}}
+{{#unless (minver "2.0.0" form.serverVersion)}}
+# note that Caddy version 1 reached end-of-life > 4 years ago back in 2020
+{{else}}
+{{#if (includes "old" form.config)}}
+# note that Caddy supports only TLSv1.2 and later
+{{else}}
```

Comment:
Will need `includes "TLSv1.1" output.protocols` instead, and the note perhaps moved to the protocols line — making a note the old config can't be supported to that extent.

Reply:
See #262.  You might suggest additions to the comments.